INSTALL.md

```diff
@@ -106,11 +107,15 @@ Builds gRPC C and C++ with boringssl.
 Please note that when using Ninja, you'll still need Visual C++ (part of Visual Studio)
 installed to be able to compile the C/C++ sources.
 ```
+> powershell git clone -b ((New-Object System.Net.WebClient).DownloadString(\"https://grpc.io/release\").Trim()) https://github.com/grpc/grpc
```

Comment:
I suggest adding `--recursive` to the initial `git clone` command instead of a separate `git submodule update --init` statement.

Reply:
Done.